lib/process_sonos_command.js

```diff
@@ -29,9 +29,12 @@ export default async function process_sonos_command(received_text) {
   } else if (received_text_lower.startsWith('spotify:')) {
     service_type = 'spotify';
     sonos_instruction = 'spotify/now/' + received_text;
-  } else if (received_text_lower.startsWith('tunein:')) {
+  } else if (received_text_lower.startsWith('tunein/')) {
     service_type = 'tunein';
-    sonos_instruction = 'tunein/now/' + received_text;
+    sonos_instruction = received_text;
```

Comment:
```suggestion
    sonos_instruction = 'tunein/play/' + received_text;
```

I guess I had this wrong before. Checking http-api docs, the correct URI string is ```tunein/play/{station ID}```.